client/src/components/desktop/ResourceDetail.jsx

```diff
@@ -84,51 +84,53 @@ function ResourceDetail(props) {
-        setZip(result.zip || '');
-        setDescription(result.description);
-        setLanguages(result.availableLanguages);
-        setCategory(result.category[0]);
-        setSubcategory(result.subcategory[0]);
-        setCost(result.cost);
+        const { res } = response;
+        setName(res.name);
+        setPhone(res.phoneNumbers);
+        setAddress(res.address ?? '');
+        setAddressLine2(res.addressLine2 ?? '');
+        setAptUnitSuite(res.aptUnitSuite ?? '');
+        setCity(res.city ?? '');
+        setState(res.state ?? '');
+        setZip(res.zip ?? '');
+        setDescription(res.description);
+        setLanguages(res.availableLanguages);
+        setCategory(res.category[0]);
+        setSubcategory(res.subcategory[0]);
+        setCost(res.cost);
         setLat(
-          result?.geoLocation == null ||
-            result?.geoLocation?.coordinates == null ||
-            Number.isNaN(result?.geoLocation?.coordinates[1])
+          res?.geoLocation === null ||
+            res?.geoLocation === undefined ||
+            res?.geoLocation?.coordinates === null ||
+            res?.geoLocation?.coordinates === undefined ||
+            Number.isNaN(res?.geoLocation?.coordinates[1])
             ? 0.0
-            : result?.geoLocation?.coordinates[1],
+            : res?.geoLocation?.coordinates[1],
         );
         setLng(
-          result?.geoLocation == null ||
-            result?.geoLocation?.coordinates == null ||
-            Number.isNaN(result?.geoLocation?.coordinates[0])
+          res?.geoLocation === null ||
+            res?.geoLocation === undefined ||
+            res?.geoLocation?.coordinates === null ||
+            res?.geoLocation?.coordinates === undefined ||
+            Number.isNaN(res?.geoLocation?.coordinates[0])
             ? 0.0
-            : result?.geoLocation?.coordinates[0],
+            : res?.geoLocation?.coordinates[0],
         );
-        setEmail(result.email || '');
-        setWebsite(result.website || '');
-        setEligibility(result.eligibilityRequirements);
-        setInternalNotes(result.internalNotes);
+        setEmail(res.email ?? '');
+        setWebsite(res.website ?? '');
+        setEligibility(res.eligibilityRequirements);
+        setInternalNotes(res.internalNotes);
         setHours(
-          result.hoursOfOperation
-            ? result.hoursOfOperation.hoursOfOperation
-            : [],
+          res.hoursOfOperation ? res.hoursOfOperation.hoursOfOperation : [],
         );
-        setRequiredDocuments(result.requiredDocuments);
-        setFinancialAidDetails(result.financialAidDetails);
-        setContacts(result.contacts);
+        setRequiredDocuments(res.requiredDocuments);
+        setFinancialAidDetails(res.financialAidDetails);
+        setContacts(res.contacts);
 
         if (authed) {
           let savedSet = new Set();
           const json = await getSavedResources();
-          savedSet = new Set(json.result);
```

Comment:
Shouldn't this still be result?

Reply:
oh ya, oops. just did search / replace for these :P